src/Microsoft.Azure.EventHubs/EventHubClient.cs

```diff
@@ -219,19 +220,26 @@ public Task SendAsync(EventData eventData, string partitionKey)
         {
             // eventDatas null check is inside ValidateEvents
             int count = EventDataSender.ValidateEvents(eventDatas, null, partitionKey);
+
             EventHubsEventSource.Log.EventSendStart(this.ClientId, count, partitionKey);
```

Comment:
Unfortunately, I have no background on eventhub.

What does it send and receive? Does it make sense to send Activity.Id, i.e. should it be encoded into payload/properties?